pkg/kubelet/kubelet_test.go

```diff
@@ -1671,30 +1671,47 @@ func getNotReadyStatus(cName string) api.ContainerStatus {
 		Ready: false,
 	}
 }
+func getReadyCondition(status api.ConditionStatus, transitionTime util.Time, reason, message string) []api.PodCondition {
+	return []api.PodCondition{{
+		Type:               api.PodReady,
+		Status:             status,
+		LastTransitionTime: transitionTime,
```

Comment:
cc @lavalamp 
Based on irl discussion with @bgrant0607 we need a way to test fields based on util.Time.
Filed https://github.com/kubernetes/kubernetes/issues/13399 to fix that.